scipy/linalg/decomp.py

```diff
@@ -778,7 +778,7 @@ def eigvals_banded(a_band, lower=False, overwrite_a_band=False,
         The bands of the M by M matrix a.
     lower : boolean
         Is the matrix in the lower form. (Default is upper form)
-    overwrite_a_band:
+    overwrite_a_band : boolean
```

Comment:
``bool`` not ``boolean``. 

The rule is actual types here (so ``int, bool, ndarray``) and clear English in the text (so ``integer, boolean, array``).